specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2106 @@
+    "definitions": {
+        "AnalyzeTextJobsInput": {
+            "allOf": [
+                {
+                    "$ref": "#/definitions/JobDescriptor"
+                },
+                {
+                    "$ref": "#/definitions/MultiLanguageBatchInput"
+                },
+                {
+                    "$ref": "#/definitions/JobManifest"
+                }
+            ]
+        },
+        "JobDescriptor": {
+            "properties": {
+                "displayName": {
+                    "description": "Optional display name for the analysis job.",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        "JobManifest": {
+            "properties": {
+                "tasks": {
+                    "description": "The set of tasks to execute on the input documents. Cannot specify the same task more than once.",
+                    "properties": {
+                        "entityRecognitionTasks": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/EntitiesTask"
+                            }
+                        },
+                        "entityRecognitionPiiTasks": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/PiiTask"
+                            }
+                        },
+                        "keyPhraseExtractionTasks": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/KeyPhraseTask"
+                            }
+                        },
+                        "entityLinkingTasks": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/EntityLinkingTask"
+                            }
+                        },
+                        "sentimentAnalysisTasks": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/SentimentAnalysisTask"
+                            }
+                        },
+                        "extractiveSummarizationTasks": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/ExtractiveSummarizationTask"
+                            }
+                        }
+                    },
+                    "type": "object"
+                }
+            },
+            "required": [
+                "tasks"
+            ],
+            "type": "object"
+        },
+        "Task": {
+            "type": "object",
+            "description": "Base task object.",
+            "properties": {
+                "taskName": {
+                    "type": "string"
```

Comment:
Add a description here, mentioning that a task name is required if multiple tasks of the same type are specified.

Reply:
If possible, also add length constraints here.